pkg/common/constant/proxy.go

```diff
@@ -19,13 +19,27 @@ package constant
 
 import "time"
 
-// default timeout 1s
+// default timeout 1s.
 const (
 	DefaultTimeoutStr = "1s"
 	DefaultTimeout    = time.Second
 )
 
-// body passthrough
+// body passthrough.
 const (
 	DefaultBodyAll = "_all"
 )
+
+// proxy code.
+const (
+	TimeoutError    = "DGP1005"
```

Comment:
Do we need the const error message or we can use 408 status code?

Reply:
agree